curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-nested-objects.md

```diff
@@ -51,6 +51,12 @@ Your code should use dot and bracket notation to access `myStorage`.
 assert(/=\s*myStorage\.car\.inside\[\s*("|')glove box\1\s*\]/g.test(code));
 ```
 
+You should not change or remove the `gloveBoxContents` declaration.
```

Comment:
I'm wondering a bit if this wouldn't be often misinterpreted as keeping the whole `const gloveBoxBontents = undefined` unchanged. With that in mind some possible alternative wording:
```suggestion
`gloveBoxContents` should still be declared with `const`.
```